fanficfare/adapters/adapter_scribblehubcom.py

```diff
@@ -270,8 +270,64 @@ def getChapterText(self, url):
         soup = self.make_soup(self._fetchUrl(url))
 
         div = soup.find('div', {'id' : 'chp_raw'})
-        if div.find('div', {'class' : 'wi_authornotes'}):
-            div.find('div', {'class' : 'wi_authornotes'}).decompose()
+
+        exclude_notes = self.getConfigList('exclude_notes')
+
+        if 'authornotes' in exclude_notes:
+            # Remove author's notes
+            # There can be multiple in one chapter, so loop until none are left
+            while div.find('div', {'class' : 'wi_authornotes'}):
+                div.find('div', {'class' : 'wi_authornotes'}).decompose()
+        else:
+            # Reformat the author's notes
+            # There can be multiple in one chapter, so loop until none are left
+            while div.find('div', {'class' : 'wi_authornotes'}):
+                author_notes = div.find('div', {'class' : 'wi_authornotes'})
```

Comment:
Here too:
`for author_notes in div.find_all('div', {'class' : 'wi_authornotes'}):`

Also, if somebody later stopped clearing `'wi_authornotes'` from `author_notes['class']` you'd end up with an infinite loop.  Best to avoid the possibility.

Ditto for the news searches below, too.